app/components/windows/EditStreamInfo.vue.ts

```diff
@@ -279,6 +284,10 @@ export default class EditStreamInfo extends Vue {
     this.userService.postFacebookPage(model.value);
   }
 
+  openFBPageCreateLink() {
+    shell.openExternal('https://www.facebook.com/pages/creation/')
```

Comment:
Let's replace this with the link to make a gaming video creator page https://www.facebook.com/gaming/pages/create